### PR TITLE
node/pkg/ethereum: improve confirmation logging

### PR DESCRIPTION
**Pull request by leoluk**
**Stack**:
- #743
- #742
- #741
- #740
- #739
- #738
- #737
- #736
- #729
- #728
- #727 ⮜

<pre>
Makes the log slightly nicer to grep when looking for a tx.
</pre>

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: node/pkg/ethereum/watcher.go [hunk 1]
```diff
@@ -296,26 +302,47 @@ func (e *Watcher) Run(ctx context.Context) error {
...
 							delete(e.pending, key)
-							ethMessagesOrphaned.WithLabelValues(e.networkName).Inc()
+							ethMessagesOrphaned.WithLabelValues(e.networkName, "not_found").Inc()
 							continue
 						}
 						if tx.BlockHash != key.BlockHash {
-							logger.Info("tx got dropped and mined in a later block; the message should have been reobserved", zap.Stringer("tx", pLock.message.TxHash),
-								zap.Stringer("block", ev.Number), zap.String("eth_network", e.networkName))
+							logger.Info("tx got dropped and mined in a different block; the message should have been reobserved",
+								zap.Stringer("tx", pLock.message.TxHash),
```
Comment (hendrikhofstadt) [inline node/pkg/ethereum/watcher.go:329]: should also log the actual blockhash

File: node/pkg/ethereum/watcher.go [hunk 2]
```diff
@@ -296,26 +302,47 @@ func (e *Watcher) Run(ctx context.Context) error {
...
+								zap.Error(err))
 							continue
 						}
 						if tx == nil {
-							logger.Info("tx was orphaned", zap.Stringer("tx", pLock.message.TxHash),
-								zap.Stringer("block", ev.Number), zap.String("eth_network", e.networkName))
+							logger.Info("tx was orphaned",
+								zap.Stringer("tx", pLock.message.TxHash),
+								zap.Stringer("blockhash", key.BlockHash),
+								zap.Stringer("emitter_address", key.EmitterAddress),
```
Comment (hendrikhofstadt) [inline node/pkg/ethereum/watcher.go:319]: should also log the actual blockhash

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]

File: node/pkg/ethereum/watcher.go (hunk 1 above)
Comment (leoluk) [reply node/pkg/ethereum/watcher.go:329]: image

Which of these?

Comment (leoluk) [reply node/pkg/ethereum/watcher.go:335]: Done

File: node/pkg/ethereum/watcher.go (hunk 2 above)
Comment (leoluk) [reply node/pkg/ethereum/watcher.go:324]: Done

**Review by evan-gray — APPROVED** [review]
tested good

**Review by hendrikhofstadt — APPROVED** [review]

**Comment by leoluk** [comment]
✓ Commit merged in pull request [#743](https://github.com/certusone/wormhole/pull/743)

**Pull request closed without merging**